test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -16,28 +16,53 @@
 
 import static org.junit.Assert.assertTrue;
 
-import org.h2.jdbcx.JdbcDataSource;
+import com.google.common.base.Strings;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
 import java.sql.Connection;
+import java.sql.DriverManager;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.ArrayDeque;
+import java.util.Properties;
 
 /** Manages an in-memory H2 database for test purposes. */
 class JdbcFixture {
-  public static final String DRIVER_CLASS = "org.h2.Driver";
-  public static final String URL =
-      "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DEFAULT_ESCAPE=";
-  public static final String USER = "sa";
-  public static final String PASSWORD = "";
+  public static final Database DATABASE;
```

Comment:
Since these are no longer static constants, drop them from UPPER_CASE to camelCase.